internal/pgbackrest/tls-server.md

```diff
@@ -37,44 +37,69 @@ to the repository host to [send and receive WAL files][archiving].
 The `pgbackrest` command acts as a TLS client and connects to a pgBackRest TLS
 server when `pg-host-type=tls` and/or `repo-host-type=tls`. The default for these is `ssh`:
 
-- https://github.com/pgbackrest/pgbackrest/blob/release/2.36/src/config/parse.auto.c#L3580
-- https://github.com/pgbackrest/pgbackrest/blob/release/2.36/src/config/parse.auto.c#L5941
+- https://github.com/pgbackrest/pgbackrest/blob/release/2.37/src/config/parse.auto.c#L3631
+- https://github.com/pgbackrest/pgbackrest/blob/release/2.37/src/config/parse.auto.c#L5997
 
 
 The pgBackRest TLS server is configured through the `tls-server-*` [options](config.md).
-In pgBackRest 2.36, changing any of these options or changing certificate contents
-requires a restart of the server.
+In pgBackRest 2.37, changing any of these options or changing certificate contents
+requires a reload of the server, as shown in the "Setup TLS Server" section of the
```

Comment:
❓ This reads to me like it does/will serve up new certs after a SIGHUP. Is that the case? (If so, :woohoo:)

Reply:
(As discussed, updating for traceability).
Yeah, that's what I saw in testing. I deleted the relevant secret and the reload was sufficient to ensure the new certificate is used.